app/src/renderer/vuex/modules/wallet.js

```diff
@@ -87,42 +85,25 @@ export default ({ node }) => {
         state.error = error
       }
     },
-    async loadDenoms({ commit, state }, maxIterations = 10) {
-      // read genesis.json to get default denoms
-
-      // wait for genesis.json to exist
-      let genesisPath = join(root, `genesis.json`)
-
-      // wait for the genesis and load it
-      // at some point give up and throw an error
-      while (maxIterations) {
-        try {
-          await fs.pathExists(genesisPath)
-          break
-        } catch (error) {
-          console.log(`waiting for genesis`, error, genesisPath)
-          maxIterations--
-          await sleep(500)
-        }
-      }
-      if (maxIterations === 0) {
-        const error = new Error(`Couldn't load genesis at path ${genesisPath}`)
-        Sentry.captureException(error)
-        state.error = error
-        return
-      }
-
-      let genesis = await fs.readJson(genesisPath)
-      let denoms = []
-      for (let account of genesis.app_state.accounts) {
-        if (account.coins) {
-          for (let { denom } of account.coins) {
-            denoms.push(denom)
-          }
-        }
-      }
+    async sendCoins(
+      { dispatch, commit, state },
+      { receiver, amount, denom, password }
+    ) {
+      await dispatch(`sendTx`, {
+        type: `send`,
+        password,
+        to: receiver,
+        amount: [{ denom, amount: amount.toString() }]
```

Comment:
Would be nice to have numbers, after readind the dispatch probably would just add unnecessary complexity